packages/fxa-payments-server/src/lib/sentry.js

```diff
@@ -129,7 +130,9 @@ SentryMetrics.prototype = {
       Sentry.init({
         release,
         dsn,
-        beforeSend,
+        beforeSend: (event) => {
+          return tagCriticalEvent(event);
+        },
```

Comment:
Nitpick: I think some of these could be shortened to just `beforeSend: tagCriticalEvent` since the signatures match?

Reply:
Cleaned those up with the short hand.